src/Wallabag/CoreBundle/Controller/EntryController.php

```diff
@@ -253,13 +252,7 @@ private function showEntries($type, Request $request, $page)
         $entries = new Pagerfanta($pagerAdapter);
 
         $entries->setMaxPerPage($this->getUser()->getConfig()->getItemsPerPage());
-        try {
-            $entries->setCurrentPage($page);
-        } catch (OutOfRangeCurrentPageException $e) {
-            if ($page > 1) {
-                return $this->redirect($this->generateUrl($type, array('page' => $entries->getNbPages())), 302);
-            }
-        }
```

Comment:
But you have to define it right? Because I guess that by default, the bundle doesn't redirect to the latest page?